packages/airnode-examples/README.md

```diff
@@ -22,7 +28,7 @@ Currently, all of the examples utilize the RRP protocol. The example RRP flow co
 2. Deploy Airnode on a cloud provider, or run locally in a docker container, and make a request using the deployed
    requester
 
-If you would like to know more about RRP, read the [API3 docs](https://docs.api3.org/airnode/v0.3/concepts/).
+If you would like to know more about RRP, read the [API3 docs](https://docs.api3.org/airnode/latest/concepts/).
```

Comment:
🙏 If this worked it would be awesome.

Reply:
It should now work ;)
There were a lot of weird edge cases in this redirects story. Deep dive into Vue.js.